### ISSUE TITLE
Unable to set title for tags in Tagset

### ISSUE THREAD

**Issue by strout18** [carbon-design-system/ibm-cloud-cognitive#2513]
## What package(s) are you using?

- [x] Carbon for IBM Products (or Carbon for IBM Cloud & Cognitive)
- [ ] Legacy/CDAI
- [ ] Legacy/Security

## Detailed description

> Describe in detail the issue you're having.

I would like to be able to have a tag within a Tagset that has a `title` attribute separate from the content of the tag (for having a tooltip that gives more detail on the tag content). This seems to be impossible with the Tag component (I found some discussion [here](https://github.com/carbon-design-system/carbon/issues/6111) ) – i.e. passing a title attribute to a non-filter Tag does nothing, so it makes sense that it also doesn't work with the TagSet. 

> Is this issue related to a specific component?

Tag and Tagset

> What did you expect to happen? What happened instead? What would you like to
> see changed?

I would like to be able to pass in a title attribute to a Tagset like so:  `      <TagSet
        align='end'
        overflowAlign='end'
        tags={[{label: 'test', title: 'More info about test'}]}
      />`
and have the title attribute be passed through to the Tag component (and its underlying `span` element) so the title shows up in the browser tooltip on hover.

> What browser are you working in?

Chrome

> What version of the @carbon/ibm-products (or @carbon/ibm-cloud-cognitive)
> package are you using?

1.13.0

> What offering/product do you work on? Any pressing ship or release dates we
> should be aware of?

Cloud pak for Watson AIOps.

## Additional information

- Screenshots or code
- Notes

**Comment by elycheea** [comment]
@strout18 We do spread the props so this may already work. Can you verify if that’s the case. If not, we can add this in.

If this is still an issue, then @qbi11y this should be pretty straightforward to add.

**Comment by strout18** [comment]
@elycheea Thanks! It does not work currently. It seems like the title prop gets thrown out if the tag is not a filter, per [here](https://github.com/carbon-design-system/carbon/issues/6111).

**Comment by strout18** [comment]
Hi @elycheea @qbi11y Any news on this issue? Thanks!

**Comment by qbi11y** [comment]
@strout18 I am taking a look at this now! WIll update soon.

**Comment by qbi11y** [comment]
@elycheea @strout18 I have given this some eyes and it looks as if there might be an issue on the Carbon side. I found this issue that seems to address the asked: [https://github.com/carbon-design-system/carbon/pull/6112](url). When I look at the storybook I do not see the prop that I see added in the issue. This ticket is a few years old and they may have removed the functionality. I am going to reach out to the channel to see if I can get more info before creating a ticket.

**Comment by qbi11y** [comment]
@elycheea @strout18 I reached out to the Carbon team to see if there was a reason that the tag hover text doesn't accept a value other than the label. There doesn't seem to be a reason, however, Michael Gower with the accessibility team thinks this isn't a great idea. It sounds like this might be a larger conversation that needs to be had before the needed updates to Carbon are made. I am going to close this out and if the discussion proves fruitful we can open a new ticket to address this.
